packages/block-editor/src/components/block-settings-menu/block-copy-button.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { MenuItem } from '@wordpress/components';
+import { useCopyToClipboard } from '@wordpress/compose';
+import { serialize } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function CopyMenuItem( { blocks, onCopy } ) {
```

Comment:
Does it make sense to partial copy from `Copy` button in Block Actions? Is this something mentally mapped to copying whole blocks? 🤔

Reply:
That's a good question. I'd love more thoughts on this, but my instinct would always be that ⌘X, ⌘C and ⌘V should match 1:1 what similar cut/copy/paste options are surfaced in the UI. I.e. be the same.

Reply:
Mmm, I think the opposite — the copy button should say "copy blocks" and work at the full block scope, like movers and transforms since it's meta to the whole block.

Reply:
> Mmm, I think the opposite — the copy button should say "copy blocks" and work at the full block scope, like duplicate, group, movers, and transforms since it's meta to the whole block.

This is my instinct as well. If so, we should ensure that focusing the Copy action highlights the whole block, just as for other block-wise actions.